shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -145,7 +146,9 @@ async fn cluster_single_rack_v3(#[case] driver: CassandraDriver) {
         };
         standard_test_suite(&connection, driver).await;
         cluster::single_rack_v3::test_dummy_peers(&connection().await).await;
-        routing::test("127.0.0.1", 9042, "172.16.1.2", 9042).await;
+
+        #[cfg(not(feature = "cassandra-cpp-driver-tests"))]
```

Comment:
why?

Reply:
Don't have tracing support on cassandra-cpp

Reply:
Oh I see, previously we were completely ignoring the driver in routing::test and thats why it used to work. :+1: